contracts/common/misc/DrainStakeManager.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.5.2;
+
+import { StakeManagerStorage } from "../../staking/stakeManager/StakeManagerStorage.sol";
+import { Lockable } from "../mixin/Lockable.sol";
+
+contract DrainStakeManager is StakeManagerStorage {
+    constructor() public Lockable(address(0x0)) {}
+
+    function drain(address destination, uint amount) external onlyGovernance {
+        require(token.transfer(destination, amount));

```

Comment:
It's a good practice to add a revert message as a second param to `require`.

Reply:
For sure, in this particular place i omitted this message for not being informative enough. I added it anyway to follow best practices.